pkg/kubelet/prober/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- tallclair
```

Comment:
anyone else from pkg/kubelet/OWNERS approvers that should be hoisted here? I think this means you'll be targeted specifically before bubbling up

Reply:
I added sig-node-approvers here, do you know if that works? Will it weight me and everyone in sig-node-approvers equally?

Reply:
I'm not sure... paging @kubernetes/sig-contributor-experience-pr-reviews in case they know

Reply:
yeah, all the folks including from the alias will have equal weight